frontend/lib/src/features/sign_up/sign_up_step_two.dart

```diff
@@ -1,134 +1,198 @@
+            text: 'Sign Up',
+            onTap: onUpdateFlowState,
+          ),
+          Padding(
+            padding: const EdgeInsets.fromLTRB(25, 30, 25, 0),
+            child: FormBuilder(
+              key: formKey,
+              autovalidateMode: AutovalidateMode.disabled,
+              onChanged: () {
+                formKey.currentState?.save();
+
+                formKey.currentState?.fields['email']?.validate(focusOnInvalid: false);
+                formKey.currentState?.fields['password']?.validate(focusOnInvalid: false);
+                formKey.currentState?.fields['password_confirmation']
+                    ?.validate(focusOnInvalid: false);
+
+                signUpController
+                  ..setPassword = formKey.currentState?.fields['password']?.value ?? ''
+                  ..setSignUpSecondButtonEnabled = formKey.currentState!.isValid;
+              },
               child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
                 children: [
-                  const Breadcrumb(
-                    text: 'Sign Up',
-                    page: 1,
+                  InputField(
+                    label: 'Username',
+                    name: 'username',
+                    initialValue: userFormData.username,
+                    validator: FormBuilderValidators.required(),
+                  ),
+                  const SizedBox(height: 26),
+                  InputField(
+                    label: 'Email',
+                    name: 'email',
+                    inputType: TextInputType.emailAddress,
+                    initialValue: userFormData.email,
+                    validator: FormBuilderValidators.compose(
+                      [
+                        FormBuilderValidators.required(),
+                        FormBuilderValidators.email(),
+                      ],
+                    ),
+                  ),
+                  const SizedBox(height: 26),
+                  InputField(
+                    label: 'Password',
+                    name: 'password',
+                    inputType: TextInputType.visiblePassword,
+                    obscureText: true,
+                    initialValue: userFormData.password,
+                    validator: FormBuilderValidators.compose([
+                      FormBuilderValidators.required(),
+                    ]),
+                  ),
+                  const SizedBox(height: 26),
+                  InputField(
+                    label: 'Confirm Password',
+                    name: 'password_confirmation',
+                    inputType: TextInputType.visiblePassword,
+                    obscureText: true,
+                    initialValue: userFormData.passwordConfirmation,
+                    validator: FormBuilderValidators.compose([
+                      FormBuilderValidators.required(),
+                      (val) {
+                        if (val != signUpController.password) {
```

Comment:
I think you can use `FormBuilderValidators.equal(signUpController.password)` here.